main/src/main/java/org/clulab/processors/clu/tokenizer/OpenDomainLexer.tokens

```diff
@@ -5,19 +5,21 @@ DATE2=4
 NUMBER=5
 FRACTION=6
 WORD=7
-TWITTER_NAME=8
-TWITTER_HASHTAG=9
-FILENAME=10
-PROGRAMMING_LANGUAGES=11
-FULLURL=12
-LIKELYURL_WWW=13
-LIKELYURL_COM=14
-EMAIL=15
-SGML=16
-HTML_CODE=17
-SMILEY=18
-LETTER_SMILEY=19
-EOS=20
-WHITESPACE=21
-SEQ_OF_UNICODES=22
-ErrorCharacter=23
+SENTENCEBREAK=8
+TWITTER_NAME=9
+TWITTER_HASHTAG=10
+FILENAME=11
+PROGRAMMING_LANGUAGES=12
+FULLURL=13
+LIKELYURL_WWW=14
+LIKELYURL_COM=15
+EMAIL=16
+SGML=17
+HTML_CODE=18
+SMILEY=19
+LETTER_SMILEY=20
+EOS=21
+WHITESPACE=22
+SEQ_OF_UNICODES=23
+ErrorCharacter=24
+'[SB]'=8
```

Comment:
This is odd, but I assume, correct.

Reply:
I.e., line 25.

Reply:
It was what Antlr generates, so I am assuming it is correct.